TA_Scheduler/utilities/AccountUtil.py

```diff
@@ -211,6 +212,13 @@ def updateAccountInfo(
         """
```

Comment:
I would update the docs to make the user aware that the method can raise a `ValidationError` if it does not validate.

Reply:
- completed suggested change
- also added docs for unvalidated method (in accounts/create.py)